samples/sample-07/src/main/resources/application.yaml

```diff
@@ -0,0 +1,17 @@
+spring:
+  docker:
+    compose:
+      lifecycle-management: start_and_stop
+      start:
+        command: up
+      stop:
+        command: down
+        timeout: 10s
+  kafka:
+    consumer:
+      bootstrap-servers: localhost:10000
+      group-id: sample07
```

Comment:
I see you use some other groups in your `@KafkaListener` methods.
Why do we still need it here?